src/libstd/io/buffered.rs

```diff
@@ -448,7 +448,16 @@ impl<R: Seek> Seek for BufReader<R> {
 /// [`flush`]: #method.flush
 #[stable(feature = "rust1", since = "1.0.0")]
 pub struct BufWriter<W: Write> {
+    // FIXME: Can this just be W, instead of Option<W>? I don't see any code
+    // paths that lead to this being None, or that ever check if it IS none,
+    // even in drop implementations. #72925.
```

Comment:
Looks like it's used to correctly implement `into_inner`. Probably it can be replaced with `ManuallyDrop`, but it isn't a trivial change. I filed #72925 so I'll get rid of this comment.